README.md

```diff
@@ -89,7 +89,7 @@ We will discuss improvements with you and agree to merge them once the [develope
 
 If you want a step-by-step walk-through on how to set-up your workspace, please check [this guideline](https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace).
 
-To compile JabRef from source, you need a Java Development Kit 21 and `JAVA_HOME` pointing to this JDK.
+To compile JabRef from source, you need a Java Development Kit 20 and `JAVA_HOME` pointing to this JDK.
```

Comment:
why do you change this?

Reply:
That should be reverted. -- No changes any more. 

Based on this proposal, I submitted https://github.com/JabRef/jabref/pull/11660